homeassistant/components/insteon_hub.py

```diff
@@ -50,6 +49,17 @@ def setup(hass, config):
         _LOGGER.error("Could not connect to Insteon service")
         return False
 
-    discovery.load_platform(hass, 'light', DOMAIN, {}, config)
+    discovery.load_platform(
+        hass,
+        'light',
+        DOMAIN,
+        {CONF_POLL: config[DOMAIN][CONF_POLL]},
+        config
+    )
+    for insteon_house in INSTEON.houses:
+        threading.Thread(
```

Comment:
What happens locally when you are running Home Assistant with this version and you press control + c ? 

I am pretty sure that it will hang indefinitely as we are not cancelling the stream, hence the thread will keep running and keep Home Assistant running.

One way to work around it is to add `daemon=True`, that way the thread will not prevent Python from shutting down and thus get killed. The more correct way would be if we could call something like `cancel_stream` when the `EVENT_HOMEASSISTANT_STOP` is fired.